sites/partners/cypress/integration/listing.spec.ts

```diff
@@ -202,4 +202,17 @@ describe("Listing Management Tests", () => {
       cy.getByID("openhouseHeader").contains("11:05:00 PM")
     })
   })
+
+  it("verify open listing warning happens", () => {
+    cy.getByTestId("listingEditButton").contains("Edit").click()
+    cy.getByTestId("nameField").type(" (Edited)")
+    cy.getByTestId("saveAndExitButton").contains("Save & Exit").click()
+    cy.getByTestId("listingIsAlreadyLiveButton").contains("Save").click()
+  })
+
+  it("verify the changes saved successfully", () => {
+    cy.fixture("listing").then((listing) => {
```

Comment:
What is this test testing? In isolation it seems like it makes sure the name is the name but doesn't do saving.